Quasardb/Native/qdb_sized_string.cs

```diff
@@ -14,6 +15,14 @@ internal unsafe struct qdb_sized_string
         internal byte* data;
         internal qdb_size_t length;
 
+        public qdb_sized_string(string str, ref GCHandle pin)
+        {
+            var content = Encoding.UTF8.GetBytes(str);
+            pin = GCHandle.Alloc(content, GCHandleType.Pinned);
```

Comment:
Out of curiosity, how is the lifecycle of this memory managed? Is this a function that tells the CLR "allocate this memory for me, and release it when appropriate" ?

Reply:
I believe the management is either when we tell it to or when the pin is destroyed.

We actively say: "Do not touch this"